tests/e2e/tests/regression/browser/stream-pending-messages.e2e.ts

```diff
@@ -0,0 +1,81 @@
+import { Chance } from 'chance';
+import { rte } from '../../../helpers/constants';
+import { acceptLicenseTermsAndAddDatabase, deleteDatabase } from '../../../helpers/database';
+import { BrowserPage, CliPage } from '../../../pageObjects';
+import {
+    commonUrl,
+    ossStandaloneConfig
+} from '../../../helpers/conf';
+
+const browserPage = new BrowserPage();
+const cliPage = new CliPage();
+const chance = new Chance();
+
+let keyName = chance.word({ length: 20 });
+let consumerGroupName = chance.word({ length: 20 });
+
+fixture `Pending messages`
```

Comment:
Is it the same test as under `critical-path`?

Reply:
the tests are different, but the feature is the same

Reply:
renamed